Sources/MapboxMaps/Style/Style.swift

```diff
@@ -416,6 +416,11 @@ public class Style {
  */
 public struct StyleTransition: Codable {
 
+    public enum CodingKeys: String, CodingKey {
```

Comment:
Can this be internal?

Reply:
This is probably not needed at all.. This struct should generate codingKeys automatically.